.github/workflows/ci.yml

```diff
@@ -124,22 +147,28 @@ jobs:
           make test
 
       - name: Pack artifacts
+        if: matrix.shard == 1
         shell: bash
         # Note that we use `cp -R` first, since that works on every platform.
         run: |
           cp -R ./build/host/sdk ./build/host/toit
-          tar -czf build/${{ matrix.artifact }} -C ./build/host -h --exclude toit/bin/mbedtls toit
+          tar -czf build/${{ steps.constants.outputs.artifact }} -C ./build/host -h --exclude toit/bin/mbedtls toit
+
       - name: Upload artifacts
+        if: matrix.shard == 1
         uses: actions/upload-artifact@v2
         with:
-          name: ${{ matrix.os }}-build
-          path: build/${{ matrix.artifact }}
+          name: ${{ runner.os }}-build
```

Comment:
This changes the name of the artifacts from:
- `linux-build` to `Linux-build`
- `macos-build` to `macOS-build`
- `windows-build` to `Windows-build`

If the artifact names are important (for example fro Jaguar), then we could change it (using the `constants` step), or we would need to adapt it.

Reply:
I don't think we use these from Jaguar at all.